library/webpack-plugins/copy-unused-files-plugin.js

```diff
@@ -6,11 +6,13 @@ const fs = require('fs-extra')
 const path = require('path')
 const walkSync = require('walk-sync')
 
+const p = 'copy-unused-files-plugin'
+
 module.exports = function copyUnusedFilesPlugin() {
   return {
     apply: compiler => {
 
-      compiler.plugin('after-emit', (compilation, done) => {
+      compiler.hooks.afterEmit.tapPromise(p, compilation => {
```

Comment:
why the `tapPromise`?

Reply:
So we can return a promise. Otherwise it would be `tapAsync` and the need to keep the `done` and the `catch` + `then` at the end of the function.

Reply:
Ah I looked over the promise, wasn't part of the diff